src/checkers/inference/solver/util/PrintUtils.java

```diff
@@ -142,4 +147,133 @@ private static void buildStatisticText(String key, Integer value, StringBuilder
         statisticsText.append(value);
         statisticsText.append("\n");
     }
+
+    public static void printUnsolvable(Collection<Constraint> mus) {
```

Comment:
what is mus? what does it mean?

Reply:
SAT specific concept - minimum unsatisfiable set. Using `mus` isn't good here, I'll change this.

Reply:
Use `unsatisfactoryConstraints`